### PR TITLE
Change the favicons for Git, Munin, Trac, and main wiki

### PR DESCRIPTION
**Pull request by Tigerfell**
openstreetmap/operations#248 discussed distinguishable favicons for subdomains of osm.org. This is a PR that aims to implement this on Git, Munin, Trac, and the main wiki. Quoting @Firefishy about the [requirements](https://github.com/openstreetmap/operations/issues/248#issuecomment-516365391)

>1. The favicons need to be easily identifiable as official OpenStreetMap.org sites. (eg: similar "Official" branding). eg: The [OSM_Tech logo](https://twitter.com/osm_tech)
>2. The actual favicons need to created for each of the sites, don't expect the Operations team to fire up inkscape to create the files.
>3. The source files need to be available, so that others could create new site versions of the favicons.

With regards to no. 3, I suggested uploading the SVG files to the wiki (alternatively to some repository). Not sure if the Git config change is correct, I assumed the file is not managed by Chef yet. 

The ticket also includes changing the icons for blog, dev, forum, help, irc, lists, and nominatim, but I did not find the relevant settings. 

(P.S. Related PR for blogs.osm.org: gravitystorm/blogs.osm.org#41.)

### CODE REVIEW

**Comment by tomhughes** [comment]
Wouldn't it have been sensible to seek feedback on the "idea" before doing a lot of work on this?

**Comment by tomhughes** [comment]
Can you clarify what exactly you have done? The ICO files aren't previewable in github so I can't easily see what they look like but I can't say I'm at all keen on the idea of changing the colour.

**Comment by Tigerfell** [comment]
> 
> 
> Wouldn't it have been sensible to seek feedback on the "idea" before doing a lot of work on this?

I did via the the ticket and talk@ mailing list (https://lists.openstreetmap.org/pipermail/talk/2019-July/082968.html). I also tried to ping ```@openstreetmap```. 

> Can you clarify what exactly you have done? 

I exchanged the current icons with the icons proposed by @zorun in https://github.com/openstreetmap/operations/issues/248#issuecomment-517991713. Additionally, I drew the first letter of the service in the upper left corner of the icon as discussed in the ticket. 
The proposed wiki logo before converted to ICO: favicon-wiki

**Comment by tomhughes** [comment]
I really don't see the point in all the garish colours - there will be about five people that will know what they mean.

The letters at least give people some chance of guessing what the meaning it.

But really trying to create unique identifiers that are meaningful and recognisable for such a large number of sites but which fit in the size of a favicon is probably madness.

**Comment by Tigerfell** [comment]
> there will be about five people that will know what they mean.

Interesting. I always thought about it more as a grouping feature like one colour belonging to the same service versus another to a different one.

File: cookbooks/git/templates/default/gitweb.conf.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@ $stylesheet = "/gitweb/static/gitweb.css";
 $logo = "/gitweb/static/git-logo.png";
 
 # the 'favicon'
-$favicon = "/gitweb/static/git-favicon.png";
+$favicon = "favicon.png";
```
Comment (Firefishy) [inline cookbooks/git/templates/default/gitweb.conf.erb]: Path should be absolute.

Comment (Tigerfell) [reply cookbooks/git/templates/default/gitweb.conf.erb]: @Firefishy As far as I can see, `/gitweb/static/git-favicon.png` is not available in this repo. So, this change would implicitly move the favicon to the repository. I have absolutely no idea how the absolute path should look. Is it something like `chef/cookbooks/git ...`?

Comment (Firefishy) [reply cookbooks/git/templates/default/gitweb.conf.erb]: I think just `$favicon = "/favicon.png";` should do it I think.

Comment (Tigerfell) [reply cookbooks/git/templates/default/gitweb.conf.erb]: Okay fixed.

**Comment by tomhughes** [comment]
Closed at openers request.

**Pull request closed without merging**